File: exercicios-4 mohammadSalim/e-bhaskara.py

A = float(input('digite o número de A: '))
B = float(input('digite o número de B: '))
C = float(input('digite o número de C: '))
Delta = ((B ** (2))- 4*A*C)

Xmais = (-B + Delta ** (1/2))/(2 * A)

Xmenos = (-B - Delta ** (1/2))/(2 * A)

print (Xmais,Xmenos)

File: Material de estudo/Java/Base/1-Primeiros conceitos de Java/E_bhaskara.java
import java.util.Scanner;

public class E_bhaskara {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o número A: ");
        double A = scanner.nextFloat();

        System.out.print("Digite o número B: ");
        double B = scanner.nextFloat();

        System.out.print("Digite o número C: ");
        double C = scanner.nextFloat();

        double Delta = ((Math.pow(B,2))- 4*A*C);

        double Xmais = (-B + Math.pow(Delta,(1/2)) )/(2 * A);

        double Xmenos = (-B - Math.pow(Delta,(1/2)) )/(2 * A);

        System.out.print("X1 = " + Xmais + ", X2 = " + Xmenos);
    }
    
}
